core/src/main/scala/org/apache/spark/sql/catalyst/expressions/ParamLiteral.scala

```diff
@@ -189,6 +188,14 @@ final class ParamLiteral(override val value: Any, _dataType: DataType, val pos:
     }
     ev.copy(initCode, isNullLocal, valueLocal)
   }
+
+  var currentValue: Any = value
+
+  override def toString: String = currentValue match {
+    case null => "null"
+    case binary: Array[Byte] => s"0x" + DatatypeConverter.printHexBinary(binary)
```

Comment:
s"" is useless here

Reply:
Yes. Just copied from Literals toString. Will remove.